x/distribution/keeper/delegation_test.go

```diff
@@ -765,72 +765,3 @@ func TestCalculateRewardsMultiDelegatorMultWithdraw(t *testing.T) {
 	// commission should be zero
 	require.True(t, app.DistrKeeper.GetValidatorAccumulatedCommission(ctx, valAddrs[0]).Commission.IsZero())
 }
-
-func Test100PercentCommissionReward(t *testing.T) {
```

Comment:
why remove this test?

Reply:
this was a new test introduced in SDK 46 - it was accidentally brought over when copying from LSM